docs/release-notes/3.0.0_Beta.md

```diff
@@ -200,3 +200,7 @@ The Agents API follows the same pattern, with a single endpoint for all agent ty
 ##### Impact
 
 Update your API calls to use the new consolidated endpoints. Review the updated API documentation in the Swagger UI of your PMM instance for the new request formats.
+
+## Fixed issues
+
+[PMM-13127](https://perconadev.atlassian.net/browse/PMM-13127) - Fixed subtab navigation not showing in breadcrumbs for **PMM Inventory** and **Settings** pages after Grafana 10 upgrade in PMM 3 Alpha.
```

Comment:
Should we show this as fix since the user wasn't impacted previously? (Alpha wasn't public, and the issue didn't exist in PMM2)

Reply:
Alpha docs are public on the [Percona Labs]https://docs.percona.com/percona-lab/ 
@rnovikovP, could you advise?

Reply:
Alpha was an internal release, so all issues "Affected 3.0 Alpha"  should be considered as "defects" and not mentioned in the Release notes.

Reply:
ok, thanks for clarifying. I've removed the mention from Rel Notes